src/mono/wasm/runtime/library_mono.js

```diff
@@ -2458,17 +2486,17 @@ var MonoSupportLib = {
 	},
 
 	mono_set_timeout: function (timeout, id) {
-		if (!this.mono_set_timeout_exec)
-			this.mono_set_timeout_exec = Module.cwrap ("mono_set_timeout_exec", null, [ 'number' ]);
+		if (!MONO.mono_set_timeout_exec)
```

Comment:
why aren't we using 'this' here?

Reply:
I tried to use `this` originally but it was throwing errors since the scoping of `this` behaves a bit differently in strict mode. Using `MONO` however works and I think it is also a bit more consistent with the rest of the function too.

Reply:
OK, but we probably need to clean up a bunch of other code in that case. Doesn't need to happen in this PR if tests are passing